src/styles/components/pods-view.less

```diff
@@ -0,0 +1,125 @@
+.pod-instances-table {
+
+  tbody {
+    line-height: 1.8;
```

Comment:
You are right, but I don't want to do this 😄  . I want it to be 80% bigger than the current line height. Check this: https://github.com/dcos/dcos-ui/blob/f2e26fef1d820f86c1bcecf172b2f11937f09a6d/src/styles/components/pods-view.less#L83